app/views/groupTab/components/TaskList.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+import React from 'react'
+import { Text, Button, Container, Content, List, ListItem } from 'native-base'
+import { connect } from 'react-redux'
+
+class TaskList extends React.Component {
+  render() {
+    const { selectedTask, setBounty } = this.props.allTasks
+    return (
+      <Container>
+        <Content>
+          <List>
```

Comment:
Can we include a fake of tasks between lines 6 and 7 and use .map over a task card component?